### PR TITLE
Centered NavBar, added hover state and changed Logo colour

### PR DESCRIPTION
**Pull request by anitaschan**

### CODE REVIEW

File: src/HomePage/NavBar/NavBar.scss [hunk 1]
```diff
@@ -10,22 +10,28 @@
...
-  justify-content: space-between;
-}
+  flex-direction: row;
+  font-family: $title-font-family;
+  font-size: 16px;
+  width: 500px;
+  justify-content: space-around;
+
+  @media (max-width: 500px) {
+    width: 450px;
```
Comment (saf94) [inline src/HomePage/NavBar/NavBar.scss]: Whats the reasoning for having this? Doesn't really affect the mobile friendliness of this navbar that much

File: src/HomePage/NavBar/NavBar.scss [hunk 2]
```diff
@@ -10,22 +10,28 @@
...
-ul {
-  li {
-    padding: 0 20px 0 20px;
-    display: inline;
-    font-family: $title-font-family;
-    font-weight: 300;
-    font-size: 16px;
+  &__link {
+    text-decoration: none;
+    color: black;
```
Comment (saf94) [inline src/HomePage/NavBar/NavBar.scss]: Im intrigued as to why the default colour of your text isn't black when all the text on your site is black 🤔

File: src/HomePage/NavBar/NavBar.scss [hunk 3]
```diff
@@ -10,22 +10,28 @@
 }
 
 .navbar {
-  margin: 10px;
+  margin: 5px auto;
   display: flex;
-  justify-content: space-between;
-}
+  flex-direction: row;
```
Comment (saf94) [inline src/HomePage/NavBar/NavBar.scss]: Is this needed? I believe the default direction is row

File: src/HomePage/NavBar/NavBar.js [hunk 4]
```diff
@@ -5,18 +5,21 @@ import { Link } from "react-router-dom";
...
-        </li>
-      </ul>
+      <Link to="/" className="navbar__link">
+        Home
+      </Link>
+      <Link to="/portfolio" className="navbar__link">
+        Portfolio
+      </Link>
+      <div>
+        <a href="/" className="Logo" />
```
Comment (saf94) [inline src/HomePage/NavBar/NavBar.js]: This isn't part of this commit but im intrigued as to why your logo has a display of flex when it has no contents inside of it, i just noticed it and thought it was weird (but wont push on it as its not part of this pr)

File: src/HomePage/NavBar/NavBar.scss (hunk 3 above)
Comment (anitaschan) [reply src/HomePage/NavBar/NavBar.scss]: Have removed

File: src/HomePage/NavBar/NavBar.scss (hunk 1 above)
Comment (anitaschan) [reply src/HomePage/NavBar/NavBar.scss]: Main reason for this was to prevent a side scroll bar from appearing on a small screen, as the navbar would push it out slightly

File: src/HomePage/NavBar/NavBar.scss (hunk 2 above)
Comment (anitaschan) [reply src/HomePage/NavBar/NavBar.scss]: Good point, I'll change it now

File: src/HomePage/NavBar/NavBar.js (hunk 4 above)
Comment (anitaschan) [reply src/HomePage/NavBar/NavBar.js]: Good point, must of left it in from experimenting before. However, just tried to remove and now it no longer appears... will have to look into this a little more

File: src/HomePage/NavBar/NavBar.scss (hunk 1 above)
Comment (saf94) [reply src/HomePage/NavBar/NavBar.scss]: Ok, i would change this to just deal with small screens properly but i dont think you should do this now. Better to fix all the mobile stuff seperately later on
